CHANGELOG.md

```diff
@@ -523,14 +526,6 @@ Non-backwards compatible changes
     - From: `∀ {x} y z → (y • x) ≈ (z • x) → y ≈ z`
     - To: `∀ x y z → (y • x) ≈ (z • x) → y ≈ z`
 
```

Comment:
I've removed the existing text regarding `Alomost*Cancellative`, in the absence of a stable story emerging about how it should be replaced. But this comment is a placeholder TODO...